schema/new_library_verification_cases.schema.json

```diff
@@ -0,0 +1,1062 @@
+                    },
+                    {
+                        "$ref": "#/$defs/AutomaticOADamperControl"
+                    },
+                    {
+                        "$ref": "#/$defs/FanStaticPressureResetControl"
+                    },
+                    {
+                        "$ref": "#/$defs/HeatRejectionFanVariableFlowControlsCells"
+                    },
+                    {
+                        "$ref": "#/$defs/ServiceWaterHeatingSystemControl"
+                    },
+                    {
+                        "$ref": "#/$defs/VAVStaticPressureSensorLocation"
+                    },
+                    {
+                        "$ref": "#/$defs/VentilationFanControl"
+                    },
+                    {
+                        "$ref": "#/$defs/WLHPLoopHeatRejectionControl"
+                    },
+                    {
+                        "$ref": "#/$defs/AutomaticShutdown"
+                    },
+                    {
+                        "$ref": "#/$defs/HeatPumpSupplementalHeatLockout"
+                    },
+                    {
+                        "$ref": "#/$defs/HeatRejectionFanVariableFlowControl"
+                    },
+                    {
+                        "$ref": "#/$defs/GuestRoomControlVent"
+                    },
+                    {
+                        "$ref": "#/$defs/GuestRoomControlTemp"
+                    },
+                    {
+                        "$ref": "#/$defs/DemandControlVentilation"
+                    }
+                ],
+                "required": [
+                    "no",
+                    "run_simulation",
+                    "simulation_IO",
+                    "expected_result",
+                    "datapoints_source",
+                    "verification_class"
+                ],
+                "additionalProperties": false,
+                "allOf": [
+                    {
+                        "$comment": "Determine when idd, weather, and ep_path are required",
+                        "if": {
+                            "properties": {
+                                "run_simulation": {
+                                    "const": true
+                                }
+                            }
+                        },
+                        "then": {
+                            "properties": {
+                                "simulation_IO": {
+                                    "required": [
+                                        "idd",
+                                        "weather",
+                                        "ep_path"
+                                    ]
+                                }
+                            }
+                        }
+                    }
+                ]
+            },
+            "minItems": 1
+        }
+    },
+    "required": [
+        "cases"
+    ],
+    "additionalProperties": false,
+    "$defs": {
```

Comment:
Would it make sense to move these to individual files for easier maintenance/expansion?

Reply:
I could see that being helpful. I think the one downside is that it makes validation a little more complex since you have to introduce a registry and (iirc) $id's. Similarly, splitting "States", "Meta", and "Imports" could be beneficial for the GUI